server/fleet/errors.go

```diff
@@ -529,3 +529,13 @@ type OrbitError struct {
 func (e OrbitError) Error() string {
 	return e.Message
 }
+
+// Message that may surfaced by the server or the fleetctl client in connection with the RunHostScript service.
+const (
+	ErrRunScriptInvalidType    = "File type not supported. Only .sh (Bash) and .ps1 (PowerShell) file types are allowed."
+	ErrRunScriptHostOffline    = "Script can’t run on offline host."
+	ErrRunScriptHostNotFound   = "Host doesn’t exist. Make sure you provide a valid hostname, UUID, osquery host ID, or node key."
+	ErrRunScriptForbidden      = "You don’t have the right permissions in Fleet to run the script."
+	ErrRunScriptAlreadyRunning = "A script is already running on this host. Please wait about 1 minute to let it finish."
+	ErrRunScriptHostTimeout    = "Fleet hasn’t heard from the host in over 1 minute. Fleet doesn’t know if the script ran because the host went offline."
```

Comment:
as a nit, the convention is to prefix an exported variable with `Err` if it's of type `error`. I understand that having the error message as a string is convenient (so you don't have to call `.Error()` everywhere, but it was a bit confusing at first.

Reply:
Thanks! This was my preliminary stab at trying to streamline error message that get used in various contexts and I'm sure there is plenty of room for improvement :)